src/errors.rs

```diff
@@ -0,0 +1,73 @@
+use crate::de;
+use crate::ser;
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+/// Error type returned from the library
+pub enum AvroError {
```

Comment:
I think it makes sense to call this `Error`, since it's already in the `avro_rs` crate. If users want to rename it the option is always available when importing.

Reply:
Yes, you are right. I started with `AvroError` to makes my `sed` easier and then I kinda rolled with it, but `Error` is the best choice.